hatchet_sdk/context/context.py

```diff
@@ -238,14 +237,17 @@ def __init__(
             self.input = self.data.get("input", {})
 
     def step_output(self, step: str) -> dict[str, Any] | BaseModel:
-        validators = self.validator_registry.get(step)
+        workflow_validator = next(
+            (v for k, v in self.validator_registry.items() if k.split(":")[-1] == step),
+            None,
+        )
```

Comment:
turned out the pydantic parsing wasn't working correctly, because the step name was something like `process1` and the key in the `validator_registry` would be something like `child:process1`. This is a hack which seems to work fine, but is there a better way to go about this?